### PR TITLE
Add PHP 8 and Contao 4.13 compatibility

### PR DESCRIPTION
**Pull request by schmidoo**
Updates the package to be used with Contao 4.13 and PHP 8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,50 +1,64 @@
...
-        "contao-community-alliance/composer-plugin": "~2.4 || ~3.0"
+    "require": {
+        "php": "^8.1",
+        "contao/core-bundle": "^4.13",
+        "contao/manager-plugin": "^2.3.1",
+        "symfony/config": "^5.4 || ^6.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0",
+        "symfony/http-foundation": "^5.4 || ^6.0",
+        "symfony/http-kernel": "^5.4 || ^6.0",
+        "symfony/routing": "^5.4 || ^6.0"
```
Comment (bytehead) [inline composer.json:30]: Are those all hard dependencies of the extension?

File: public/css/.htaccess [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<IfModule !mod_authz_core.c>
```
Comment (bytehead) [inline public/css/.htaccess:1]: Can be deleted (file)

File: public/js/.htaccess [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<IfModule !mod_authz_core.c>
```
Comment (bytehead) [inline public/js/.htaccess:1]: Can be deleted (file)

**Review by bytehead — CHANGES_REQUESTED** [review]
Please eliminate the `Fbi` subfolder in the namespace and filesystem.

File: composer.json (hunk 1 above)
Comment (schmidoo) [reply composer.json]: Yes, except for `symfony/routing`. Has been removed 👍

Comment (bytehead) [reply composer.json]: 👍

File: composer.json [hunk 4]
```diff
@@ -1,50 +1,63 @@
...
         {
-            "name":"David Greminger",
-            "email":"dg@1up.io",
-            "homepage":"http://1up.io",
-            "role":"Developer"
+            "name": "David Greminger",
+            "email": "dg@1up.io",
+            "homepage": "http://1up.io",
+            "role": "Developer"
         }
```
Comment (bytehead) [inline composer.json:20]: Missing your part :)

**Review by bytehead — CHANGES_REQUESTED** [review]

**Review by bytehead — APPROVED** [review]
LGTM 🥳

**Comment by bytehead** [comment]
Thank you!

**Pull request merged**